src/App.tsx

```diff
@@ -1,11 +1,13 @@
 import './App.scss';
 
 import SignUp from 'components/Signup';
+import { NavigationBar } from 'components/NavigationBar';
```

Comment:
If you create index.ts in the components folder and add lines:
export * from "./Signup"
export * from "./NavigationBar"

then you can write 
import {SignUp, NavigationBar} from "components"

Reply:
fixed